src/Test.UnitTests.Sarif.Converters/TestData/ClangTidy/ExpectedOutputs/ValidResults.sarif

```diff
@@ -0,0 +1,272 @@
+{
+  "$schema": "https://schemastore.azurewebsites.net/schemas/json/sarif-2.1.0-rtm.5.json",
+  "version": "2.1.0",
+  "runs": [
+    {
+      "results": [
+        {
+          "ruleId": "clang-diagnostic-error",
+          "message": {
+            "text": "too many errors emitted, stopping now"
+          },
+          "locations": [
+            {
+              "physicalLocation": {
+                "artifactLocation": {
+                  "uri": "",
+                  "index": 0
+                },
+                "region": {
+                  "charOffset": 0
+                }
+              }
+            }
+          ]
+        },
+        {
+          "ruleId": "readability-convert-member-functions-to-static",
+          "message": {
+            "text": "method 'print' can be made static"
+          },
+          "locations": [
+            {
+              "physicalLocation": {
+                "artifactLocation": {
+                  "uri": "/home/lsp/projects/staticlib/src/Hello.cpp",
```

Comment:
>home/lsp/projects/staticlib/src/Hello.cpp [](http://example.com/codeflow?start=27&length=41)

check other converters to see if we normally use the artifacts property or if we use the artifactLocation.

right now, you are duplicating the information

Reply:
you mean the uri is not needed here because the url is already in the artifacts section, since we have the index here, we can know the url by looking at artifacts section by index right.

this is same as LintConverter, it is also duplicating this info,

the index is automaticlly added by code, once we specifc the url here. I think ideally the code that  automaticlly add index can delete the url at the same time, if we want to do that. let me know if we want make changes.